tests/libvmi/vmi.go

```diff
@@ -125,8 +126,8 @@ func WithSEV() Option {
 	}
 }
 
-func baseVmi(name string) *kvirtv1.VirtualMachineInstance {
-	vmi := kvirtv1.NewVMIReferenceFromNameWithNS("", name)
+func baseVmi(namespace, name string) *kvirtv1.VirtualMachineInstance {
```

Comment:
The VMI spec should not set the namespace.
This is a repeating mistake.

Tge namespace is to br set by the creation call, which will return a vmi that includes the ns.

Reply:
gotcha, I did this because some tests started failing because the VMI had no namespace set, I guess they relied on this.

I'll fix this.

https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/7551/pull-kubevirt-e2e-k8s-1.21-sig-compute/1513783827359600640